packages/@lwc/ssr-compiler/src/__tests__/fixtures.spec.ts

```diff
@@ -64,14 +62,22 @@ function testFixtures() {
             pattern: '**/index.js',
         },
         async ({ filename, dirname, config }) => {
-            const compiledFixturePath = await compileFixture({
-                input: filename,
-                dirname,
-            });
-
             const errorFile = config?.ssrFiles?.error ?? 'error.txt';
             const expectedFile = config?.ssrFiles?.expected ?? 'expected.html';
 
+            let compiledFixturePath;
+            try {
+                compiledFixturePath = await compileFixture({
+                    input: filename,
+                    dirname,
+                });
+            } catch (err: any) {
+                return {
+                    [errorFile]: `SSR compilation error:\n${err.message}`,
```

Comment:
```suggestion
                    [errorFile]: err.message,
```

This seems redundant since the filename already contains `ssr`.